### ISSUE TITLE
Add all widgets to the widgets showcase

### ISSUE THREAD

**Issue by joshka** [ratatui/ratatui-website#249]
The new widgets showcase is up (https://github.com/ratatui-org/website/pull/245) and a few widgets are already done. But we need a small demo for each of:

- [x] Block
- [x] BarChart
- [x] Calendar
- [x] Canvas
- [x] Chart
- [x] Gauge
- [x] LineGauge
- [x] List
- [x] Paragraph
- [x] Scrollbar
- [x] Sparkline
- [x] Table
- [x] Tabs

For info on how to contribute, see the [Widget showcase project README](https://github.com/ratatui-org/website/blob/main/code/widget-showcase/README.md). Feel free to ask questions about this here or in [Discord](https://discord.gg/pMCEU9hNEj)

**Comment by Valentin271** [comment]
What about something like that for the chart? Representing some kind of statistical classification? (with more data, real data could be cool too like for the world map on the canvas) Or would it fit the examples better?
![image](https://github.com/ratatui-org/website/assets/36198422/9dd9ad47-b90b-476b-94e2-1b3d04e43ea6)

**Comment by joshka** [comment]
Sounds good. Should just be a matter of changing the data points and re running the vhs tape.

**Comment by Its-Just-Nans** [comment]
after #798, this issue is closeable right ?

**Comment by joshka** [comment]
yup
